tools/ctor_evaller.py

```diff
@@ -125,7 +125,7 @@ def add_func(asm, func):
 var stackMax = stackTop + totalStack;
 if (stackMax >= totalMemory) throw 'not enough room for stack';
 
-var dynamicTop = stackMax;
+var dynamicTopPtr = stackMax;
```

Comment:
@kripken: This is actually something I'm not completely certain - what should ctor_evaller do about `DYNAMICTOP_PTR` here?